apps/wallet-mobile/src/features/Settings/RemoveWallet/RemoveWalletScreen.tsx

```diff
@@ -66,7 +66,9 @@ export const RemoveWalletScreen = () => {
           </WalletInfo>
         </ScrollView>
 
-        <Actions>
+        <Spacer fill />
+
+        <View style={styles.actions}>
```

Comment:
can put back <Actions? (less overhead for readability)

Reply:
done